src/resources/unlimitedCompany.ts

```diff
@@ -0,0 +1,91 @@
+import { ResourceIF } from '@/interfaces'
+import { NameRequestTypes, RouteNames } from '@/enums'
+import { CorpTypeCd, GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
+
+export const UnlimitedCompanyResource: ResourceIF = {
+  entityType: CorpTypeCd.BC_ULC_COMPANY,
+  displayName: GetCorpFullDescription(CorpTypeCd.BC_ULC_COMPANY),
+  title: 'Unlimited Company Statement',
+  description: null,
+  statement: null,
+  nameRequestType: NameRequestTypes.CC,
```

Comment:
Should be UL.

Reply:
You betcha, commented on this above (somewhere)